<launch>

<node pkg="camera_nbg" type="camera_nbg_node" name="camera_nbg_node" output="screen">
      <remap from="rgb/image"       to="/camera_/rgb/image_rect_color"/>
      <remap from="depth/image"     to="/camera_/depth_registered/image_raw"/>
      <remap from="rgb/camera_info" to="/camera_/rgb/camera_info"/>
</node>

</launch>
